DevelopGitHub/GitHubExtension/GitHubExtension.Activity.External.Tests/TestsForRoutes/Extensions/ExternalActivityRouteTestExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using GitHubExtension.Activity.External.WebAPI;
+
+namespace GitHubExtension.Activity.External.Tests.TestsForRoutes.Extensions
+{
+    public static class ExternalActivityRouteTestExtensions
+    {
+        public static string ForGetGitHubActivityRoute(this string url, int page = 1)
```

Comment:
Hmm...Then it's my fault that I passed such extension methods. Can you refactor this?